worker-languagedetection/README.md

```diff
@@ -41,6 +59,13 @@ e.g. DetectedLanguage1\_ConfidencePercentage:100
 
 - DetectedLanguages_ReliableResult : A boolean flag that signals whether the result is reliable. 
 
+#### Multi-Field Processing:
+
+- `WORKER_LANG_DETECT_FIELD_PREFIX$FieldName`: 
+   This field will contain two values at present, the first value in this string will be the percentage of the detected language, e.g. `98%`. The second value in the string after a space will be the language that was detected language code, e.g. `en`. If a field is multi value this result will be a combined result for all of the values.
+
+**NOTE:** This fields value may be extended in future, between each value within this string will be a space separating the result values. In future confidence percentage could also be added.
```

Comment:
Use the actual text specified in the JIRA ticket:
Document the following *_"In future we may expand the field value to include whether the result is unreliable so it is recommended to split on spaces and take the second value to retrieve the language code. Any additions to the value will include a space before the new value."_*